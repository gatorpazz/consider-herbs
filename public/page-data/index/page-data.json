{"componentChunkName":"component---src-pages-index-js","path":"/","result":{"data":{"contentfulSiteSettings":{"heroTitle":"Get back to your nature.","heroBio":"Consider the holistic power of herbs and experience the healthy effects of our earth.","callToActionButton":"Explore Today"}},"pageContext":{}},"staticQueryHashes":[]}