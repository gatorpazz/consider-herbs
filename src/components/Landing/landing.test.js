import React from "react"
import renderer from "react-test-renderer"
import { PureLanding as Landing } from "./index"

describe("Landing", () => {
  it("renders correctly", () => {
    const data = {
      contentfulSiteSettings: {
        id: "8ad82eb0-2504-51d6-b2f5-03808afda33d",
        heroTitle: "Get back to your nature.",
        heroBio: "Consider the holistic power of herbs and experience the healthy effects of our earth.",
        callToActionButton: "Explore Today",
        feature1Headline: "Feature 1 Text Here",
        feature1Text: "You won't believe all the things that feature one has in store. Check out this icon above me.",
        feature2Headline: "Feature 2 Text Goes Here",
        feature2Text: "It's really quite unbelievable. You won't believe all the things that feature two has in store. ",
        feature3Headline: "Feature 3 Will Be Shown",
        feature3Text: "Can you feel it? That's the power of herbs healing your body. Really, it's true.",
        heroImage: {
          file: {
            url: "//images.ctfassets.net/l7aye13px8h4/1vq9ylwI0cK5ArUt0FMfL0/dba43fd73de8413901048c44897fe41d/IMG_0264.jpg"
          }
        },
        feature1Image: {
          file: {
            url: "//images.ctfassets.net/l7aye13px8h4/4BEnbw0GIPZWYwbkNEAsjN/cb15eaee66260373fd55b278aa8e85ce/IMG_0291.jpg"
          }
        },
        feature2Image: {
          file: {
            url: "//images.ctfassets.net/l7aye13px8h4/6851HOwp5dLgiVuOzcapz5/7a64695affa6be8fc6bae619444e355d/IMG_8249__1_.jpg"
          }
        },
        feature3Image: {
          file: {
            url: "//images.ctfassets.net/l7aye13px8h4/63DM5qkSAdP8Lp3wyoZf1C/7fcd9a4466284f68fdf342429a230f93/IMG_0311.jpg"
          }
        }
      }
    }

    const url = "TEST123";
    
    const tree = renderer
      .create(<Landing data={data} url={url} />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})