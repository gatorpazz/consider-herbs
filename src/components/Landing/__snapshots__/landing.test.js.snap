// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Landing renders correctly 1`] = `
<div>
  <div
    className="py-20"
  >
    <div
      className="container mx-auto px-8 grid xl:grid-cols-7 gap-4"
    >
      <div
        className="xl:col-span-3 sm:text-center xl:text-left"
      >
        <h1
          className="text-4xl tracking-tight font-bold text-gray-900 sm:text-5xl md:text-6xl"
        >
          Get back to your nature.
        </h1>
        <h2
          className="font-semibold mt-3 text-base text-gray-600 sm:mt-5 sm:text-lg md:mt-5 md:text-xl "
        >
          Consider the holistic power of herbs and experience the healthy effects of our earth.
        </h2>
        <a
          href="/shop"
        >
          <button
            className="mt-5 text-white font-bold bg-green-500 py-3 px-5 shadow rounded-lg tracking-wider"
          >
            Explore Today
          </button>
        </a>
      </div>
      <div
        className="xl:col-span-4"
      >
        <img
          alt="hero"
          src="//images.ctfassets.net/l7aye13px8h4/1vq9ylwI0cK5ArUt0FMfL0/dba43fd73de8413901048c44897fe41d/IMG_0264.jpg"
        />
      </div>
    </div>
  </div>
  <div
    className="container mx-auto px-8 lg:flex grid-cols-3"
  >
    <div
      className="lg:w-1/3 mx-4 mb-12 bg-gray-50 rounded-xl shadow-md overflow-hidden"
    >
      <div
        className="md:flex"
      >
        <div
          className="p-8"
        >
          <h3
            className="font-bold text-3xl mb-4"
          >
            Feature 1 Text Here
          </h3>
          <img
            alt=""
            src="//images.ctfassets.net/l7aye13px8h4/4BEnbw0GIPZWYwbkNEAsjN/cb15eaee66260373fd55b278aa8e85ce/IMG_0291.jpg"
          />
          <p
            className="font-semibold text-xl"
          >
            You won't believe all the things that feature one has in store. Check out this icon above me.
          </p>
        </div>
      </div>
    </div>
    <div
      className="lg:w-1/3  mb-12 mx-4 bg-gray-50 rounded-xl shadow-md overflow-hidden"
    >
      <div
        className="md:flex"
      >
        <div
          className="p-8"
        >
          <h3
            className="font-bold text-3xl mb-4"
          >
            Feature 2 Text Goes Here
          </h3>
          <img
            alt=""
            src="//images.ctfassets.net/l7aye13px8h4/6851HOwp5dLgiVuOzcapz5/7a64695affa6be8fc6bae619444e355d/IMG_8249__1_.jpg"
          />
          <p
            className="font-semibold text-xl"
          >
            It's really quite unbelievable. You won't believe all the things that feature two has in store. 
          </p>
        </div>
      </div>
    </div>
    <div
      className="lg:w-1/3 mb-12 mx-4 bg-gray-50 rounded-xl shadow-md overflow-hidden"
    >
      <div
        className="md:flex"
      >
        <div
          className="p-8"
        >
          <h3
            className="font-bold text-3xl mb-4"
          >
            Feature 3 Will Be Shown
          </h3>
          <img
            alt=""
            src="//images.ctfassets.net/l7aye13px8h4/63DM5qkSAdP8Lp3wyoZf1C/7fcd9a4466284f68fdf342429a230f93/IMG_0311.jpg"
          />
          <p
            className="font-semibold text-xl"
          >
            Can you feel it? That's the power of herbs healing your body. Really, it's true.
          </p>
        </div>
      </div>
    </div>
  </div>
  <div
    className="container mx-auto"
  >
    <div
      className="mb-12 mx-4 bg-gray-50 rounded-xl shadow-md overflow-hidden  text-center"
    >
      <div
        className="p-8"
      >
        <div
          className="px-2 -mt-6"
        >
          <div
            className="text-center"
          >
            <h1
              className="text-2xl md:text-3xl text-grey-800 font-medium leading-loose my-3 w-full"
            >
              Join our Mailing List!
            </h1>
            <div
              className="w-full text-center"
            >
              <div
                className="max-w-sm mx-auto p-1 pr-0 flex items-center"
              >
                <input
                  className="flex-1 appearance-none rounded shadow p-3 text-grey-dark mr-2 focus:outline-none"
                  placeholder="email@example.com"
                  type="email"
                />
                <button
                  className="bg-green-600 text-white text-base font-semibold rounded-md shadow-md hover:bg-green-600 p-3"
                  onClick={[Function]}
                  type="submit"
                >
                  Sign Up
                </button>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
